src/sharedHooks/useAuthenticatedInfiniteQuery.ts

```diff
@@ -8,7 +8,7 @@ import { useCurrentUser } from "sharedHooks";
 // Should work like React Query's useInfiniteQuery with our custom reactQueryRetry
 // and authentication
 const useAuthenticatedInfiniteQuery = (
-  queryKey: Array<string>,
+  queryKey: QueryKey,
```

Comment:
maybe this type "correction" is just highlighting an erroneous caller of this code but.... we spread this here https://github.com/inaturalist/iNaturalistReactNative/pull/3252/files#diff-76b47bbee105159db467d028840da68c4b5dd0dabe8a57db2fb2fce94a9a1f28R21.

Just looking at the later change https://github.com/inaturalist/iNaturalistReactNative/pull/3252/files#diff-9059d29458ca045e30c8ebcbc99e0c41e7317961c0c2a47c70a6c9c2b36051f3L7-R7

```
  queryKey: Array<string>,
  queryKey: QueryKey[],
```

we go form `string[]` to `QK[]`. Do you have any insight as to if we're using this wrong in either place?